src/components/main/main.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { Todo as OneTodo } from '../../todo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const Main: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      {todos.map((todo: Todo) => {
+        return <OneTodo todo={todo} key={todo.id} />;
```

Comment:
why you need return ?
```suggestion
        return <Todo todo={todo} key={todo.id} />;
```